testcases/open_posix_testsuite/conformance/interfaces/pthread_cancel/3-1.c

```diff
@@ -20,30 +20,32 @@
 #include "safe_helpers.h"
 
 #define TIMEOUT_MS 5000
-#define SLEEP_MS 1
+#define SLEEP_MS 1000
```

Comment:
This looks wrong. 
It suggests that we sleep for a 1000 ms, but it's actually 1000 nanoseconds.
It changes sleep from 1ms to 1us, however it's counted towards thread_sleep_time as an entire second.